stonesoup/hypothesiser/tests/test_distance.py

```diff
@@ -35,3 +41,26 @@ def test_mahalanobis(predictor, updater):
 
     # The hypotheses are sorted correctly
     assert min(hypotheses, key=attrgetter('distance')) is hypotheses[0]
+
+
+def test_distance_include_all(predictor, updater):
+
+    timestamp = datetime.datetime.now()
+    track = Track([GaussianState(np.array([[0]]), np.array([[1]]), timestamp)])
+    detection1 = Detection(np.array([[2]]))
+    detection2 = Detection(np.array([[3]]))
+    detection3 = Detection(np.array([[10]]))
+    detections = {detection1, detection2, detection3}
+
+    measure = measures.Mahalanobis()
+    hypothesiser = DistanceHypothesiser(
+        predictor, updater, measure=measure, missed_distance=1,
+        include_all=True)
+
+    hypotheses = hypothesiser.hypothesise(track, detections, timestamp)
+
+    # There are 4 hypotheses - Detections and Missed Detection
+    assert len(hypotheses) == 4
+
+    # And not detection3
```

Comment:
Looks like this comment got copied directly from another test and does not accurately reflect what is happening here.